cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedExpr.qll

```diff
@@ -1493,6 +1491,75 @@ class TranslatedAssignExpr extends TranslatedNonConstantExpr {
   }
 }
 
+class TranslatedBlockAssignExpr extends TranslatedNonConstantExpr {
+  override BlockAssignExpr expr;
+
+  final override TranslatedElement getChild(int id) {
+    id = 0 and result = this.getLeftOperand()
+    or
+    id = 1 and result = this.getRightOperand()
+  }
+
+  final override Instruction getFirstInstruction() {
+    // The operand evaluation order should since block assignments behave like memcpy.
```

Comment:
This comment reads a bit weird. Did you mean "[...] should be left-to-right [...]"?

Reply:
Woops:
```suggestion
    // The operand evaluation order should not matter since block assignments behave like memcpy.
```

Reply:
Does that make more sense?

Reply:
That makes a lot of sense, yes!